dhcpv4/option_relay_agent_information.go

```diff
@@ -12,16 +20,16 @@ type RelayOptions struct {
 
 var relayHumanizer = OptionHumanizer{
 	ValueHumanizer: func(code OptionCode, data []byte) fmt.Stringer {
-		return OptionGeneric{data}
+		return raiValue{val: data}
 	},
 	CodeHumanizer: func(c uint8) OptionCode {
-		return GenericOptionCode(c)
+		return raiSubOptionCode(c)
 	},
 }
 
 // String prints the contained options using Relay Agent-specific option code parsing.
 func (r RelayOptions) String() string {
-	return r.Options.ToString(relayHumanizer)
+	return "\n" + r.Options.ToString(relayHumanizer)
```

Comment:
Move the line break to the raiValue String method. I think it will make it more consistent with:
https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/options.go#L47

Reply:
I don't get it, the raiValue is the sub-option value, the line break need to be added in beginning of sub-option list, so if we want to be consistent  https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/options.go#L47, then shouldn't it be changing OptionHumanizer.Stringify() like following?
```
func (oh OptionHumanizer) Stringify(code uint8, data []byte) string {
	c := oh.CodeHumanizer(code)
	val := oh.ValueHumanizer(c, data)
        // if the value string contains "\n" then it is an option contains sub-option like option82,  insert a line break
	if strings.Contains(val.String(), "\n") {
		return fmt.Sprintf("%s: \n%s", c, val)
	}
	return fmt.Sprintf("%s: %s", c, val)
}
```

Reply:
> I don't get it, the raiValue is the sub-option value, the line break need to be added in beginning of sub-option list

Can't it be added at the end of the raiValue instead?

Reply:
I still don't get it, the purpose is to add a newline between "Relay Agent Information:" and sub-option list like following:
```
    Relay Agent Information:**<newline here>** 
        Agent Circuit ID Sub-option: cid-1 ([99 105 100 45 49])
        Agent Remote ID Sub-option: rid-1 ([114 105 100 45 49]) 
```
how adding a newline at the end of raiValue help to achieve that?

Reply:
I mean at the beginning:

`return fmt.Sprintf("\n%s (%v)", string(rv.val)), rv.val)`

Reply:
that doesn't work since raiValue is the the value of the suboption inside option82, e.g. it is "cid-1 ([99 105 100 45 49])" part of "Agent Circuit ID Sub-option: cid-1 ([99 105 100 45 49])", add a newline at the beginning will look like this:
```
    Relay Agent Information:     Agent Circuit ID Sub-option:
cid-1 ([99 105 100 45 49])
        Agent Remote ID Sub-option:
rid-1 ([114 105 100 45 49])
```

Reply:
Right, ok. Disregard this comment.